scripts/Frequency_Domain_Analysis.py

```diff
@@ -8,11 +8,25 @@
 from __future__ import (absolute_import, division, print_function)
 
 from Muon.GUI.FrequencyDomainAnalysis.frequency_domain_analysis_2 import FrequencyAnalysisGui
-import PyQt4.QtCore as QtCore
+from qtpy import QtCore
 from save_python import getWidgetIfOpen
 
 
 Name = "Frequency_Domain_Analysis_2"
+"""
```

Comment:
Is this an error message?

Reply:
ah, I need to delete this. Thank you for spotting it